lib/open_api_spex/cast_parameters.ex

```diff
@@ -30,7 +30,8 @@ defmodule OpenApiSpex.CastParameters do
     object_schema = %Schema{
       type: :object,
       properties: properties,
-      required: required
+      required: required,
+      additionalProperties: false
```

Comment:
This is ok for now, but when headers are casted (sometime in the future), we'll want to allow additional properties by default.